mockapihelpers.go

```diff
@@ -6,6 +6,7 @@ import (
 	api "k8s.io/client-go/pkg/api/v1"
 )
 
+// MockAPIHelpers for testing support
```

Comment:
This file is auto-generated. Making manual changes to auto-generated files is not recommended. Please revert this change.